iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/primary/processing/oozie_app/workflow.xml

```diff
@@ -91,6 +91,13 @@
 			<name>input_project_concept</name>
 			<description>input project concept directory</description>
 		</property>
+		<!-- project reference extraction related -->
+		<property>
+            <name>referenceextraction_project_fundingclass_blacklist_regex</name>
+            <value>$^</value>
+            <description>regexp matching funding class, describes projects which 
+            should be blacklisted by reference extraction algorithm</description>
+        </property>
```

Comment:
Fixed. Just realized another problem: I've set tab=4 spaces in eclipse while auto-formatting indents with tabs...